coordinator/src/models/errors/validation-errors.ts

```diff
@@ -58,10 +58,23 @@ export class MedicationRequestIncorrectValueError implements ValidationError {
 export class MessageTypeError implements ValidationError {
   message = `MessageHeader.eventCoding.code must be one of '${
```

Comment:
Maybe not for this PR, but would it be nicer to use a switch statement to handle different message types, and just drop through to an 'Unsupported Message Type' sort of error as the default case?

Reply:
yea would be good to reduce the amount of places i have to change for a new message type :+1: